docs/open-source-tizen/tools/gbs/gbs-build.md

```diff
@@ -264,160 +275,155 @@ info: Local repo can be found here:
+The source package file (`src.rpm`) contains everything needed to recreate a specific version of a package. This makes the `src.rpm` file a great way to distribute source code. However, it is optional when developing source code, especially when the source Git tree is huge.
 
-Adding --skip-srcrpm option will enable GBS to skip the building of src.rpm file, thus speeding up the building process of huge source git trees during development phase. An example is shown below:
+Adding the `--skip-srcrpm` option enables GBS to skip the building of the `src.rpm` file, speeding up the building process of huge source Git trees during development:
 
 ```bash
 $ cd <Path_to_crosswalk>
 $ gbs build -A i586 --skip-srcrpm
 ```
 
-#### Use distributed compiler networks
-
-Though GBS has already provided --threads option to speed up the build process by activating multiple build workers, yet for huge git tree like crosswalk, the efficiency of just using multiple build workers in one local machine is far from satisfactory. That's where --icecream option comes in.
+### Using Distributed Compiler Networks
 
---icecream option activates distributed compiler networks, that is, GBS will use build workers on both local machine and distributed networks, thus further speeding up the build process.
+Though GBS provides the `--threads` option to speed up the build process by activating multiple build workers, the efficiency of just using multiple build workers in 1 local machine is far from satisfactory, especially for a huge Git tree, such as crosswalk.
 
-An example is shown below:
+To improve the build process efficiency further, use the `--icecream` option, which activates distributed compiler networks. The option makes GBS use build workers on both the local machine and distributed networks.
 
 ```bash
 $ gbs build -A i586 --icecream=10
 ```
 
-### Fetch the project build conf and customize build root (for Advanced Users)
+## Fetching the Project Build Conf and Customizing the Build Root (for Advanced Users)
 
-Project build conf describes the project build configurations for the project, including pre-defined macros/packages/flags in the build environment. In Tizen releases, the build conf is released together with the released repo. You can find an example at: [http://download.tizen.org/releases/daily/trunk/ivi/latest/builddata/xxx-build.conf](http://download.tizen.org/releases/daily/trunk/ivi/latest/builddata/xxx-build.conf)
+The project build conf describes the build configurations for the project, including all macros, packages, and flags predefined in the build environment. In Tizen releases, the build conf is released together with the released repository. For an example, see [http://download.tizen.org/releases/daily/trunk/ivi/latest/builddata/xxx-build.conf](http://download.tizen.org/releases/daily/trunk/ivi/latest/builddata/xxx-build.conf).
```

Comment:
Link is broken

Reply:
IVI profile is legacy. 
I think you don't change it.

Reply:
OK